### DISCUSSION TITLE
webdav挂载问题

### DISCUSSION THREAD

**Discussion by chennyangit** [AlistGo/alist#5648] [General]
有些webdav客户端比如RaiDrive、MT管理器、X-plore能正常挂载alist，有些却返回401 Unauthorized比如FolderSync、青盐云听。这些客户端挂载坚果云的webdav就没事

**Comment by pujiaxun** [comment]
同样遇到，请问有解决吗？

**Reply by chennyangit** [reply]
可能是ssl证书兼容性的问题，换成通配符证书后已解决。推荐在ohttps申请免费通配符证书，记得3个月更新一次，用起来很舒服~

**Reply by pujiaxun** [reply]
> 配

我最后试了几次，协议、域名、路径啥的，最后发现保存的配置里没有帐号名，怀疑可能是安卓 1password 填充问题...

**Comment by mymine** [comment]
+1
